tabs/configuration.js

```diff
@@ -511,20 +511,20 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
 
             function save_serial_config() {
                 if (semver.lt(CONFIG.apiVersion, "1.6.0")) {
-                    MSP.send_message(MSP_codes.MSP_SET_CF_SERIAL_CONFIG, MSP.crunch(MSP_codes.MSP_SET_CF_SERIAL_CONFIG), false, save_misc);
+                    MSP.send_message(MSP_codes.MSP_SET_CF_SERIAL_CONFIG, MSP_crunch(MSP_codes.MSP_SET_CF_SERIAL_CONFIG), false, save_misc);
```

Comment:
The MSP_crunch function needs to be further rework. The function itself is however just a static function and only serialize the input data in a structured way. In my opinion it is not striclty specific to MSP, I have some ideas how to further improve this.